### PR TITLE
V2 final fixes

### PR DESCRIPTION
**Pull request by James-Byrne**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>TpEmberShop</title>
+  <title>TP Demo Store</title>
```
Comment (marjaimate) [inline app/index.html]: We should use the brand TestingPays here

File: app/index.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" href="#">Testing Pays Shop!</a>
+        <a class="navbar-brand" href="#">Testing Pays Demo Store!</a>
```
Comment (marjaimate) [inline app/index.html]: TestingPays, one word

Comment (James-Byrne) [reply app/index.html]: @marjaimate Those are fixed now

Comment (James-Byrne) [reply app/index.html]: @marjaimate just found a bug, let me fix that one first

**Pull request merged**